Client/Application/Migration.swift

```diff
@@ -7,11 +7,18 @@ import Shared
 import BraveShared
 import SwiftKeychainWrapper
 import Data
+import BraveRewards
 
 private let log = Logger.browserLogger
 
 class Migration {
-    private(set) public static var braveCoreBookmarksMigrator: BraveCoreMigrator?
+    
+    private(set) public static var braveCoreSyncObjectsMigrator: BraveCoreMigrator?
+
+    private(set) public static var isChromiumMigrationCompleted: Bool = {
+        return Preferences.Chromium.syncV2BookmarksMigrationCompleted.value &&
+            Preferences.Chromium.syncV2HistoryMigrationCompleted.value
+    }()
```

Comment:
Doesn't need to be an assignment:

```suggestion
    public static var isChromiumMigrationCompleted: Bool {
        Preferences.Chromium.syncV2BookmarksMigrationCompleted.value &&
            Preferences.Chromium.syncV2HistoryMigrationCompleted.value
    }
```

Reply:
Fix at https://github.com/brave/brave-ios/pull/3703/commits/7b2245d8faec9661c2c7682df4c0f91cc229d06c